keadm/cmd/keadm/app/cmd/debug/check.go

```diff
@@ -1,3 +1,16 @@
+/*
```

Comment:
We don't need to add copyright for previous files, only add 2024 copyright for new file : )